types/openui5/sap.ui.table.d.ts

```diff
@@ -1366,13 +1362,6 @@ declare module "sap/ui/table/Column" {
      * Default value is `false`.
      */
     getAutoResizable(): boolean;
-    /**
```

Comment:
this method should not disappear in 1.100

Reply:
Okay, it's in line with OpenUI5 change https://github.com/SAP/openui5/commit/8af8fdb59809da0d7bf538425a91ff771efbd294 .